components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -240,8 +240,8 @@ class SettingsPage extends React.Component<Props, State> {
         return (
           <ModalRedirect
             id={'redirect-modal-id-verification-required'}
+            errorText={getLocale('redirectModalKYCRequiredText').replace('$0', getWalletProviderName(externalWallet))}
```

Comment:
As far as `IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_*`, I wanted to reserve `$0` for the name of the provider and reserve `$1` to indicate that the string contains markup, so that I can split it up based on `$1` and `$2`, as in:
```
<message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_WALLET_OWNERSHIP_VERIFICATION_FAILURE_TEXT" desc="">There was a problem validating your browser's request to connect your Brave Rewards wallet to your <ph name="PROVIDER">$0<ex>Uphold</ex></ph> account. You can <ph name="BEGIN_LINK">$1</ph>contact support<ph name="END_LINK">$2</ph> for more help.</message>
```

Reply:
Sure thing, done!